management/templates/index.html

```diff
@@ -370,8 +370,10 @@ <h4 class="modal-title" id="errorModalTitle"> </h4>
   api_credentials = ["", ""];
   if (typeof localStorage != 'undefined')
     localStorage.removeItem("miab-cp-credentials");
+    localStorage.removeItem("miab-cp-token");
```

Comment:
I don't think this can work? you are setting the cookie to be `HttpOnly` which means it won't be accessible client-side.

Reply:
True. Probably need a server route like /bye or /logout for this.